bioptim/interfaces/acados_interface.py

```diff
@@ -105,30 +107,49 @@ def __set_constrs(self, ocp):
         ## TODO: implement constraints in g
 
         # path control constraints
+        self.x_bound_max = []
+        self.x_bound_min = []
+        d = [-1, 0, 1]
```

Comment:
I'd rather prefer following bioptim's logic. i.e. removing the d vector and working with the following idx : 0 stands for start, 1 for intermediate and 2 for end